bin/circleci.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
-export DOCKER_VERSION=${1:-1.9.1}
+export DOCKER_VERSION=${1:-1.9.1} # only used by Circle 1.0
 export BUILD_HARNESS_PROJECT=${2:-build-harness}
-export BUILD_HARNESS_BRANCH=${3:-master}
+export BUILD_HARNESS_BRANCH=${3:-circle-2.0-wip}
```

Comment:
Is this deliberate?

Reply:
Just for testing, will revert this for realz

Reply:
Just while testing